index.html

```diff
@@ -34,5 +34,175 @@ <h1 class="greetings"> Hey there <br>
+                  <li class="text-java">JavaScript</li>
+                  <li class="text-html">HTML</li>
+                </ul>
+                <div class="button1"><button class="button-text" type="button">See project </button></div>
+              </div>
+          </div>
+          </section>
+            <section class="about-container">
+              <div class="text-aline">
+                </div>
+                <div class="text-about">
+                  <div class="plane-container">
+                    <h3>About me</h3>
+                    <div class="plane-image">
+                      <img src="images/plane.svg" alt="plane">
+                    </div>
+                  </div>
+                  
+                    <div>
+                      <p class="h6"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                    </div> 
+                  
+                    <div class="button2">
+                      <button class="button-resume" type="button">Get my resume </button>
+                    </div>
+                </div>
+                <div class="plane">
+                  <img src="images/square.svg" alt="square">
+                </div>
+              </div>
+               <div class="section6">
+                <article class="quote">
+                  <div class="holder-text">
+                    <img src="images/holder.svg" alt="holder">
+                    <div class="holder-text1">
+                      <img src="images/ellipse.svg" alt="holder">
+                      <ul>
+                        <li class="Language">
+                          <p>Languages</p>
+                          </li>
+                            <ul class="media">
+                              <li>JavaScript</li>
+                              <li>Ruby</li>
+                              <li>HTML</li>
+                              <li>CSS</li>
+                            </ul>
+                      </ul>              
+                    </div>
+                  </div>
+                  </article>
+                    <section class="quote">
+                        <div class="holder-text">
+                        <img src="images/holder.svg" alt="holder">
+                        <div class="holder-text1">
+                          <img src="images/ellipse2.svg" alt="holder">
+                          <ul>
+                            <li class="Language">
+                              <p>Frameworks</p>
+                            </li>
+                            <ul class="media">
+                              <li>React.js</li>
+                              <li>Ruby on Rails</li>
+                              <li>RSpec</li>
+                              <li>Capybara</li>
+                              <li>Selenium</li>
```

Comment:
![techs](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first/assets/99656687/d00e74ba-f627-403e-b4bc-834399949a8e)
- [ ] Please follow the Figma design to style the technology list. This will improve your projects aesthetics 👍🏾